torch/quantization/fx/quantize.py

```diff
@@ -599,6 +599,19 @@ def __init__(self):
         # mapping from matched node to activation_post_process
         # must be filled before convert
         self.activation_post_process_map = None
+        # mapping from node name to qconfig that should be used for that node
```

Comment:
please hold off this one, this is refactored in https://github.com/pytorch/pytorch/pull/43901 and we no longer need to initialize them in __init__

Reply:
sure, sounds good

Reply:
I read through the relevant sections of https://github.com/pytorch/pytorch/pull/43901/files, not sure I understand your suggestion.  Can you give some more context?

this PR is not adding or changing any functionality, it just improves documentation.  It's not intuitive to have a Python class where important state is not declared in `__init__`.

Reply:
I see, if this is just used as declaration then it might be OK, but we'll need to make sure in convert we distinguish the declared and defined states correctly

Reply:
sg.  I think for people who are not the original code authors but who need to understand/modify the code, glancing at `__init__` to see what the important state is for a class is pretty common.  This should make it easier for others to quickly understand the high level without having to read every LOC.